src/KubernetesClient.Classic/KubernetesClient.Classic.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net48</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net48;netstandard2.1;net6.0</TargetFrameworks>
```

Comment:
why netstandard2.1 and net6 here?
classic will support netstandard2.0 only

to use net6, please use main sdk

Reply:
have removed the netstandard 2.1 as didn't provide benefit.